.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.6.x, 3.7.x, 3.8.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Upgrade pip
+        run: pip install --upgrade pip
+      - name: Install Poetry
+        run: pip install poetry
+      - name: Cache virtual environment
+        uses: actions/cache@v2
+        with:
+          path: .venv
+          key: ${{ runner.os }}-python-${{ matrix.python-version }}-venv-${{ hashFiles('**/poetry.lock') }}
+      - name: Install dependencies
+        run: poetry install --no-interaction
+      - name: pytest and coverage
+        run: poetry run pytest --cov=mimesis --cov-report=xml
+      - name: mypy
+        run: poetry run mypy mimesis/ tests/
+      - name: pyroma
+        run: poetry run pyroma -d .
+      # - name: Upload coverage to Codecov
+      #   uses: codecov/codecov-action@v1
+      #   with:
+      #     token: ${{ secrets.CODECOV_TOKEN }}
+      #     file: ./coverage.xml
+      #     env_vars: OS,PYTHON
+      #     fail_ci_if_error: true
+
+  deploy:
+    if: ${{ github.ref == 'ref/head/master' }}
```

Comment:
Do I understand correctly that this will trigger a new release each time when changes merged to master?

Reply:
Yes this workflow is triggered by the `push` event but the `deploy` job will only run if the `build` job was successful AND when merging to master.

https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows

We could setup a separate workflow file that handles the publishing if required.

Reply:
Isn't a better solution to use tags for publishing new releases?

Reply:
@lk-geimfari yes I changed, check new changes.

Reply:
Thanks! I'll merge this PR in the morning. Thank you again!